src/msgpack.cpp

```diff
@@ -63,16 +90,31 @@ MSGPACK_API_VERSION_NAMESPACE(MSGPACK_DEFAULT_API_NS)
                 break;
             }
             case msgpack::type::BIN: {
+                // For backwards compatibility
```

Comment:
This is not helpful, all previously generated MXRs would fail inside `from_value()` because of file version mismatch.

Reply:
Yes it would fail in `from_value`, but with an error about the incompatible version, which I think is a much more informative error message.